components/ParameterForm.vue

```diff
@@ -168,6 +184,14 @@ const formData = ref(
   // or to the previous scenario's values if any.
   appStore.currentScenario.parameters ? { ...appStore.currentScenario.parameters } : initialiseFormDataFromDefaults(),
 );
+
+// Making the vue select searchable means that it's possible to unset a parameter value (to undefined) if you clear the search
+// input. In this case we just want to be able to revert to the previous value it had. However, this is tricky as we
+// don't get the previous value in any watch of formData since it's watching deep changes in an object, and we can't
+// use a computed setter for values in an object type. So here we keep a copy of the last full dictionary and reset in the watch
+// if required.
+const previousFullFormData = ref({ ...formData.value });
```

Comment:
This is obviously quite gross. Potentially avoidable by allowing the select drop down to be unset and using the invalids to show a validation error on submit - however that's not trivial since we're not using a core ui input, which gives all the nice validation Ui and styling. Could also just not make the select searchable, but that
does seem useful for country because there are so many options...

Reply:
I wasn't able to make the select drop down set a parameter to unset when I removed this code, how can this be reproduced?